templates/core/about/builds.html

```diff
@@ -7,72 +7,75 @@
 	</p>
 
-	<p>
-		The README of a crate is taken from the <code>readme</code> field defined in
-		<code>Cargo.toml</code>. If a crate doesn't have this field,
-		no README will be displayed.
-	</p>
+	<h3>Notes on using Docs.rs</h3>
 
-	<h3>Diagnosing a failed build</h3>
-
-	<h4>Missing dependencies</h4>
+	<h4>Setting a README</h4>
 	<p>
-		Missing dependencies are a common reason for a failed build.
-		Docs.rs dependencies are managed through
-		<a href="https://github.com/rust-lang/crates-build-env">crates-build-env</a>;
-		see <a href="https://forge.rust-lang.org/docs-rs/add-dependencies.html">Forge</a> for instructions
-		on how to make a PR. You can always <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>
-		if you're having trouble.
+		The README of a crate is taken from the <code>readme</code> field defined in
+		<code>Cargo.toml</code>. Omitting this field means that no README will be displayed.
 	</p>
 
 	<h4>Detecting Docs.rs from <code>build.rs</code></h4>
 	<p>
-		Docs.rs builds crates with the environment variable <code>DOCS_RS</code> set to
-		<code>1</code>, which enables the crate to detect docs.rs and build the
-		crate differently. This can be helpful if you need
+		Docs.rs sets the environment variable <code>DOCS_RS</code> to <code>1</code>.
+		Detecting Doc.rs and building your crate differently can be helpful if you need
 		dependencies for building the library, but not for building the documentation.
 	</p>
 
 	<h4>Detecting Docs.rs from <code>#[cfg]</code> attributes</h4>
 	<p>
-		You can detect Docs.rs by having a <a href="https://doc.rust-lang.org/cargo/reference/features.html">feature</a>
-		which is only set by Docs.rs. See <a href="metadata">Metadata</a> for more information.
+		Docs.rs sets a <a href="https://doc.rust-lang.org/cargo/reference/features.html">feature</a>. See <a href="metadata">Metadata</a> for details.
+	</p>
+
+	<h4>Testing documentation builds locally</h4>
+	{%- set build_subcommand = docsrs_repo ~ "/blob/master/README.md#build-subcommand" -%}
+	<p>
+		The Docs.rs <a href="{{ build_subcommand | safe }}">README</a> describes how to build
+		unpublished crate documentation locally using the same build environment as the Docs.rs build agent.
 	</p>
 
-	<h4>Global sandbox limits</h4>
+	<h3>Diagnosing failed builds</h3>
 
+	<h4>Missing dependencies</h4>
 	<p>
-		All the builds on docs.rs are executed inside a sandbox with limited
-		resources. The current limits are the following:
+		Missing dependencies are a common reason for failed builds.
+		Docs.rs dependencies are managed through
+		<a href="https://github.com/rust-lang/crates-build-env">crates-build-env</a>.
+		See <a href="https://forge.rust-lang.org/docs-rs/add-dependencies.html">Forge</a> for instructions
+		on how to make a PR. You can always <a href="{{ docsrs_repo | safe }}/issues">file an issue</a>
+		when having trouble.
 	</p>
 
-	{{ macros::crate_limits(limits=limits) }}
+	<h4>Writing files to read-only directories</h4>
```

Comment:
Crate authors are _not_ writing to the read-only directories ;)
```suggestion
	<h4>Read-only directories</h4>
```